MonoGame.Framework/Desktop/OpenTKGamePlatform.cs

```diff
@@ -88,13 +88,13 @@ class OpenTKGamePlatform : GamePlatform
 		private OpenALSoundController soundControllerInstance = null;
         // stored the current screen state, so we can check if it has changed.
         private bool isCurrentlyFullScreen = false;
-        private Toolkit toolkit;
         private int isExiting; // int, so we can use Interlocked.Increment
+
+        int windowDelay = 11;
```

Comment:
Just to make myself a bit clearer, it's unrelated to this error, it's a 5th error... yey